.github/workflows/test.yml

```diff
@@ -110,3 +129,21 @@ jobs:
       env:
         AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
         AWS_SECRET_KEY: ${{ secrets.AWS_SECRET_KEY }}  
+
+    # Update GitHub status for dispatch events
+    - name: "Update GitHub Status for this ref"
+      uses: docker://cloudposse/github-status-updater
+      #if: ${{ github.event_name == 'repository_dispatch' }}
+      env:
```

Comment:
with:
        # We need to use args because GitHub actions overwrites many `GITHUB_*` environment variables
        args: "-action update_state -ref ${{ github.event.pull_request.head.sha }}"